### PR TITLE
Add running multiple rules

### PR DESCRIPTION
**Pull request by adamkvitek**
# Customers are still asking about running multiple rules simultaneously

Slack: https://r2c-community.slack.com/archives/CK86BJ5DW/p1646752265856209?thread_ts=1646750695.175009&cid=CK86BJ5DW

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [ ] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

**Comment by adamkvitek** [comment]
Hello there @ievans, I was not sure about the flags used in: https://r2c-community.slack.com/archives/CK86BJ5DW/p1646752265856209?thread_ts=1646750695.175009&cid=CK86BJ5DW

I mean what is:  semgrep --config=r/gitlab.bandit.B307 --config=p/secrets - what is =r/ and =p/ (I am sorry that my command line knowledge is limited here, if this is a common thing, feel free to send me a link to some external source of information.)

File: docs/running-rules.md [hunk 1]
```diff
@@ -70,7 +70,13 @@ Create a YAML rule file that you can run repeatedly.
     semgrep --config path/to/rule.yaml
     ```
 
-### Appendix
+### Running multiple rules simultaneously
+
+To run multiple rule files simultaneously, use `--config` before every YAML URL, or Semgrep registry entry name. See the following code example (substitute the colored values as necessary):
```
[the remark this answers is not in the document]
Comment (ievans) [reply docs/running-rules.md:75]: Yeah, I think it's better to just say "to run multiple rules simultaneously..."

**Comment by ievans** [comment]
@adamkvitek not sure how well those are documented but:

p/xyz = "pack", as seen on for instance https://semgrep.dev/p/xss
r/x is explained here: https://github.com/returntocorp/semgrep-rules#how-do-i-use-these-rules

File: docs/running-rules.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ievans) [reply docs/running-rules.md:75]: Yeah, I think it's better to just say "to run multiple rules simultaneously..."

File: docs/cli-usage.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ Options:
                                   URL will be used to log in to the Semgrep
                                   registry.
 
-                                  To run multiple rule files simultaneously, use --config
+                                  To run multiple rules simultaneously, use --config
```
Comment (adamkvitek) [inline docs/cli-usage.md:83]: TODO note for me - change this in Semgrep codebase as well.

**Pull request merged**